ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-ospf.j2

```diff
@@ -120,20 +120,29 @@ router ospf {{ process_id.id }}
 {%     endif %}
 {%     if process_id.redistribute.static is defined %}
```

Comment:
This could / should also be replace with
```suggestion
{%     if process_id.redistribute.static is arista.avd.defined %}
```

Reply:
As discussed, `is defined` must be kept for now since `static:` can be `null`.